tool/automation/framework/app/src/main/kotlin/com/jetbrains/teamcity/common/constants/ValidationConstants.kt

```diff
@@ -5,4 +5,5 @@ package com.jetbrains.teamcity.common.constants
  */
 object ValidationConstants {
     const val ALLOWED_IMAGE_SIZE_INCREASE_THRESHOLD_PERCENT = 5.0f
+    const val PRE_PRODUCTION_IMAGE_PREFIX = "EAP"
```

Comment:
For which cases do we need it?

Reply:
@burnasheva it was added in order to exclude new `EAP` images inside of staging registry from the comparison, having an assumption we don't treat it as a specific release. 
A related filtering is applied within [this line](https://github.com/JetBrains/teamcity-docker-images/pull/41/files#diff-159f861e7ed104470463497ea1d14cf82d0b4f070911f315148bfc53d73d5f8dR99).

Reply:
I see, thank you. We stopped publishing EAP tag several years ago, but if it's only for filtering I don't see anything bad in having it here.